CONTRIBUTING.md

```diff
@@ -205,7 +205,7 @@ target PR branch you can do so by setting the PR_TARGET_BRANCH environment varia
 PR_TARGET_BRANCH=origin/example sbt validatePullRequest
 ```
 
-If you have already run all tests and now just need to check that everything is formatted and or mima passes there
+If you have already run all tests and now just need to check that everything is formatted and mima passes. There
 are a set of `all*` commands aliases for running `test:compile` (also formats), `mimaReportBinaryIssues`, and `validateCompile` 
```

Comment:
I agree the previous version wasn't great, but in the new version the first sentence is not complete. What do you think about something like:

```suggestion
If you already ran all tests and just need to check formatting and mima, there
is a set of `all*` command aliases that run `test:compile` (also formats), `mimaReportBinaryIssues`, and `validateCompile` 
```